projects/python-client/src/datapane/client/api/dp_object.py

```diff
@@ -187,13 +187,6 @@ def delete(self):
         self.res.delete()
         log.debug(f"Deleted object {self.url}")
 
-    def update(self, **kwargs):
```

Comment:
Sorry - I missed this earlier, but I don't think this method should be removed, it's a generic method to perform a `patch` http request to the backend

Reply:
No worries, I also missed this! I've added it back in and the tests are happy now